freshbooks/client.py

```diff
@@ -307,6 +308,16 @@ def invoice_profiles(self) -> AccountingResource:
     def items(self) -> AccountingResource:
         """FreshBooks items resource with calls to get, list, create, update, delete"""
         return AccountingResource(self._client_resource_config(), "items/items", "item", "items")
+    
+    @property
+    def journal_entries(self) -> AccountingResource:
+        """FreshBooks items resource with calls to get, list, create, update, delete"""
+        return AccountingResource(self._client_resource_config(), "journal_entries/journal_entries", "journal_entry", "journal_entries")
```

Comment:
Looks like you can only create journal entries, so `missing_endpoints=["list", "get", "update", "delete"]` should be added.